### PR TITLE
Implements small labels to abductor glands to see what they do. Part 2.

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

Abductors and observers will now be able to see what organ's do upon shift-clicking them. Uhhh, mothblocks was racist towards Australians and the AEST so I GUESS I had to remake this.

## Why It's Good For The Game

Uhhh, who the fuck thought having to check the wiki for every organ was a good idea? This allows for a better experience for abductor scientists and curious ghosts wanting to see what happens next.

## Changelog

:cl:
qol: Adds descriptions to abductor glands upon shiftclick. 
/:cl:

### CODE REVIEW

**Review by tralezab — CHANGES_REQUESTED** [review]
add periods to the end of the descriptions

**Comment by MrMelbert** [comment]
image
arm baba, the examine message itself has the period

**Comment by carshalash** [comment]
> add periods to the end of the descriptions

Done on purpose, leaving the sentence with two periods ended with double periods due to true_name var.

image

File: code/modules/antagonists/abductor/equipment/glands/spider.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/item/organ/heart/gland/spiderman
-	true_name = "araneae cloister accelerator"
+	true_name = "araneae cloister accelerator. Makes the abductee exhale spider pheromones and spawn spiderlings"
```
Comment (MrMelbert) [inline code/modules/antagonists/abductor/equipment/glands/spider.dm:2]:
```suggestion
	true_name = "araneae cloister accelerator. The abductee occasionally exhales spider pheromones and will spawn spiderlings"
```

File: code/modules/antagonists/abductor/equipment/glands/transform.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /obj/item/organ/heart/gland/transform
-	true_name = "anthropmorphic transmorphosizer"
+	true_name = "anthropmorphic transmorphosizer. Makes the abductee occasionally change appearance and species"
```
Comment (MrMelbert) [inline code/modules/antagonists/abductor/equipment/glands/transform.dm:2]:
```suggestion
	true_name = "anthropmorphic transmorphosizer. The abductee will occasionally change appearance and species"
```

File: code/modules/antagonists/abductor/equipment/glands/viral.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /obj/item/organ/heart/gland/viral
-	true_name = "contamination incubator"
+	true_name = "contamination incubator. Makes the abductee carrier of a random advanced disease - abductee is NOT affected"
```
Comment (MrMelbert) [inline code/modules/antagonists/abductor/equipment/glands/viral.dm:2]:
```suggestion
	true_name = "contamination incubator. The abductee becomes a carrier of a random advanced disease - of which they are unaffected by"
```

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Honestly, just for readability on the code side + to sooth armhulen it may be better to
- move the period from the examine block to the tips themselves
- rename `true_name` to something like `abductor_hint` or something

**Comment by carshalash** [comment]
Periods fixed for arm, did joey's suggestions too.

File: code/modules/antagonists/abductor/equipment/gland.dm [hunk 4]
```diff
@@ -6,7 +6,7 @@
 	status = ORGAN_ROBOTIC
 	organ_flags = NONE
 	beating = TRUE
-	var/true_name = "baseline placebo referencer"
+	var/abductor_hint = "baseline placebo referencer"
```
Comment (MrMelbert) [inline code/modules/antagonists/abductor/equipment/gland.dm:9]: Auto-doc this since you're here anyways?

**Review by MrMelbert — CHANGES_REQUESTED** [review]
One last nit-pick

**Pull request merged**